flashinfer/jit/cute_dsl_core.py

```
@@ -184,6 +184,36 @@ def build(self) -> None:
+        with FileLock(self.lock_path, thread_local=False):
+            if (
+                self.module_dir.exists()
+                and _read_meta(self.meta_path) != self.expected_meta
+            ):
+                logger.info(
+                    f"Invalidating stale CuTe-DSL module {self.module_dir_name}"
+                )
+                shutil.rmtree(self.module_dir, ignore_errors=True)
+            try:
+                self._export()
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)